lib/src/network/mod.rs

```diff
@@ -143,6 +144,7 @@ pub enum BackendStatus {
 
 #[derive(Debug,PartialEq,Eq)]
 pub struct Backend {
+  pub id:                 Uuid,
```

Comment:
It could. By regular integer do you mean an increasing integer ? It just felt like easier with UUID